conf/application.conf

```diff
@@ -18,31 +18,21 @@
 
 include "frontend.conf"
 
-play.http.session.cookieName = "catalogue"
```

Comment:
As discussed, this needs to be the same as internal-auth-frontend - it should be set globally for the enviroment (app-config-common/platops) - but the default (mdtp) should suffice